web-back/utils.ts

```diff
@@ -13,6 +13,8 @@ export function jsonResponse(
 
 export const isValidUrl = (url: string) => {
     try {
+        if (url === "")
+            return true;
```

Comment:
Your function is**Valid**Url returns `true` when the url is empty.
Please return false and add curly braces on the same occasion

Reply:
Following specification, an user can decide to not put any image or thumbnail thus meaning, the url to verify is empty. If I want to follow the specifications, I either have to rollback the last commit to accept an empty string or keep these 2 lines.

Reply:
Oh you're right; fine then :+1: